SingularityService/src/main/java/com/hubspot/singularity/scheduler/SingularityNewTaskChecker.java

```diff
@@ -300,6 +300,7 @@ private CheckTaskState getTaskState(SingularityTask task) {
   private Optional<CheckTaskState> checkLbState(BaragonRequestState lbState) {
     switch (lbState) {
       case SUCCESS:
+      case INVALID_REQUEST_NOOP:
         return Optional.of(CheckTaskState.HEALTHY);
```

Comment:
And if we get it back from an ADD request, I can't see the client handling that as nominal